tests/philips_hue_test.py

```diff
@@ -51,57 +51,114 @@ def test_set_color(philips_hue_client, requests_mock):
     assert "{'success': {'/lights/1/state/hue': '0'}}" in response.text
 
 
-def test_trigger_from_incident_invalid_state(philips_hue_client, requests_mock):
-    notification = {"incident": {"state": "unknown"}}
+def test_trigger_from_incident_bad_url(philips_hue_client, requests_mock):
     bridge_ip_address = philips_hue_client.bridge_ip_address
     username = philips_hue_client.username
     matcher = re.compile(f'http://{bridge_ip_address}/api/{username}')
     requests_mock.register_uri('PUT', matcher,
                                text=philips_hue_mock.mock_hue_put_response)
 
+    with pytest.raises(philips_hue.BadAPIRequestError) as e:
+        assert philips_hue_client.set_color('2', 65535)
+    assert str(e.value) == 'invalid Philips Hue url'
+
+
+def test_trigger_from_incident_invalid_state():
+    policy_name = 'unknown_policy'
+    incident_state = 'unknown'
+    notification = {'incident': {'policy_name': policy_name, "state": incident_state}}
+    policy_hue_mapping = {
+        'default': {
+            'open': 65280,
+            'closed': 24432
+        }
+    }
+
     with pytest.raises(philips_hue.UnknownIncidentStateError) as e:
-        assert philips_hue.trigger_light_from_monitoring_notification(
-            philips_hue_client, notification, 1)
-    assert 'Incident state must be "open" or "closed"' in str(e.value)
+        assert philips_hue.get_target_hue_from_monitoring_notification(
+            notification, policy_hue_mapping)
+
+    expected_error_value = ("Incident state for Google Cloud alerting policy "
+                            "'unknown_policy' must be one of: ['open', "
+                            "'closed']; actual: 'unknown'")
+    assert str(e.value) == expected_error_value
+
+
+def test_trigger_hue_with_nondefault_open_incident():
```

Comment:
This is actually testing get_target_hue_from_monitoring_notification now, so rename accordingly, here and elsewhere?

Reply:
Thanks for pointing this out, just made the changes. I'll definitely be more careful in the future and make sure I properly update my test cases when changing the structure of my code.